### ISSUE TITLE
Remove UI

### ISSUE THREAD

**Issue by GCHQDeveloper314** [gchq/gaffer-tools#1046]
The UI fails to meet our Accessibility policy. It is not practical to maintain it and it needs to be removed in Gaffer 2.

**Comment by GCHQDeveloper314** [comment]
Closed by #1048
